project-koku/koku
# Adding start/end dates

Fixes [COST-2759](https://issues.redhat.com/browse/COST-2759)

Changes proposed in this PR:
* Adds start end dates to cache_args to enable multiple month simultaneous processing
*

Testing Instructions:
1. start up with multiple koku workers - Add scale=2 (number of workers) to your up command
2. Ingest data spanning multiple months
3. see that each worker picks up a different month for summary regardless if a different month is currently in progress

---
Additional Context:
